recipe/install-gcc.sh

```diff
@@ -116,7 +116,7 @@ done
 exec gcc $fl ${1+"$@"}
 EOF
 
-  chmod 755 ${PREFIX}/bin/c{8,9}9
+  chmod 755 ${PREFIX}/bin/${CHOST}-c{8,9}9
```

Comment:
Should we put back links to c89 and c99 here to help downstream users? Right now I am guessing this might break some builds? Maybe only do this on the native compilers?

Reply:
I don't think that's necessary. We don't have `gcc` and only `${CHOST}-gcc`.

Reply:
OK. We did have these in bins on previous releases so this is a breaking changed. Happy to punt though. They should have always been prexifed by CHOST anyways.